### ISSUE TITLE
Abruptly stops with exit code 136, without any logs.

### ISSUE THREAD

**Issue by kamaleshtangudu** [aws/aws-for-fluent-bit#231]
Docker containers are killed without any error logs, just with exit code 136. Running 2.19.0 on a container in AWS Fargate and this happens randomly without any spikes in any metrics for the container. Seems it could be an issue with internal dependency fluent-bit:1.8.3 but it would be great if we could have some logs.

**Comment by zhonghui12** [comment]
Hi @kamaleshtangudu, we just cut a new release which includes Fluent Bit 1.8.6. Could you please try our latest version to see if the issue still exists?  Thanks.

**Comment by zhonghui12** [comment]
Tracking the same issue in our upstream and I could see this comment said upgrading the version would help: https://github.com/fluent/fluent-bit/issues/4035#issuecomment-910439268. Please let us know if our latest version could fix your issue.

**Comment by nokute78** [comment]
Are you using elasticsearch plugin ?
If so, https://github.com/fluent/fluent-bit/issues/3905 may be the same issue.

I sent a patch https://github.com/fluent/fluent-bit/pull/3931 and it was merged to fluent-bit:v1.8.4.
https://fluentbit.io/announcements/v1.8.4/

**Comment by zhonghui12** [comment]
Hi @kamaleshtangudu, any update here? From upstream, fluent bit >1.8.5 fixes this issue so please upgrade your aws-for-fluent-bit version and see if the problem is resolved. Thanks

**Comment by kamaleshtangudu** [comment]
Yup the issue got resolved I moved to version(2.19.1) which uses fluent bit version  1.8.6 and this got resolved. Thanks for all the support.
